soc/arm/st_stm32/stm32h7/power_m7.c

```diff
@@ -48,8 +48,15 @@ static int stm32_power_init(const struct device *dev)
 	LL_PWR_ConfigSupply(LL_PWR_LDO_SUPPLY);
 #endif
 	LL_PWR_SetRegulVoltageScaling(LL_PWR_REGU_VOLTAGE_SCALE1);
+
+	/*
+	 * At this point it should actually only be necessary to wait for ACTVOSRDY during startup.
```

Comment:
Might use some rewording here: "it should actually only be necessary to wait for ACTVOSRDY".
That's not what we're doing as we're waiting for both VOS and ACTVOSRDY.